apps/web/src/lang/fr.json

```diff
@@ -7372,5 +7372,13 @@
   "wMp4x6": {
     "defaultMessage": "Télécharger le guide (EN)",
     "description": "Link text for English guidance resource download"
+  },
+  "9vQj8o": {
+    "defaultMessage": "Oh! on dirait que vous avez été trop vite!",
```

Comment:
Sorry, this looked a little funny, and while not a sure thing, seems the lowercase is not common unless a comma?
https://context.reverso.net/translation/french-english/Oh!+on+dirait+que

Reply:
Happy to see this marked out of scope as a concern, tho!

Reply:
@gobyrne thought's on this one?